spdx-utils/src/main/kotlin/SpdxUtils.kt

```diff
@@ -77,9 +77,16 @@ fun calculatePackageVerificationCode(files: List<File>, excludes: List<String> =
  * Return the SHA-1 sum of the given file as hex string.
  */
 private fun sha1sum(file: File): String {
-    file.inputStream().use {
+    file.inputStream().use { inputStream ->
+        val buffer = ByteArray(4 * 1024 * 1024)
```

Comment:
Could you please add a code comment how you determined the buffer size? 4 MiB seems rather small to me. Even if it's rather arbitrary, please state that in the comment.

Reply:
Done.